Dockerfile

```diff
@@ -15,6 +15,9 @@ RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://repo.radeon.com/rocm/apt/6.0
 # From docs.amd.com for installing rocm. Needed to install properly
 RUN sh -c "echo 'Package: *\nPin: release o=repo.radeon.com\nPin-priority: 600' > /etc/apt/preferences.d/rocm-pin-600"
 
+# rocgdb doesn't work on 22.04, workaround by installing the older python packages that are in 20.04
+RUN add-apt-repository -y ppa:deadsnakes/ppa
```

Comment:
What's the connection between rocgdb and python version ? 
rocgdb uses python3.8 internally ?

Reply:
Yea, it uses libpython3.8 directly which is not available on ubuntu 22.04.

Reply:
You need to install "software-properties-common" before running add-apt-repository

Reply:
I added the missing package.